frontend/views/reports.js

```diff
@@ -85,6 +133,7 @@ module.exports = Backbone.View.extend({
         this.$el.html(template(context));
         this.resultsView = new ReportTableView({
```

Comment:
Well, there are 2 different view objects involved here. The one here is for the whole page, and then it creates a new view object just for the results Table (and now for the charts) and inserts it inside this one.